### DISCUSSION TITLE
Calling an URL for granted tags

### DISCUSSION THREAD

**Discussion by tschuster89** [omersiar/esp-rfid#600] [Q&A]
I using this as an addon for my doorpi system and would like to call an url if a granted tag is detected.

This url calls an event/action at my doorpi system which opens the door. Any ideas how to get this?

At the moment the only way that may works is to use mqtt and home assistant which calls the url by an virtual switch

Thanks a lot, Tim

**Comment by matjack1** [comment]
hey @tschuster89 yes, I would have done the same. Use MQTT and HA to call a URL, I think it's a good approach.

Out of curiosity, are you using the `dev` branch of the code? Thank you!
